src/ketu/async/sink.clj

```diff
@@ -140,44 +141,35 @@
                :ketu.sink/producer producer
                :ketu.sink/abort-input abort-input}
 
+        threads-done-chan        (async/chan)
+        auto-close-chan          (async/chan)
```

Comment:
`auto-close-chan` can be deleted entirely. See comment below.

Reply:
Removed.